tools/talisman/talisman-precommit.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+preCommitFile='.git/hooks/pre-commit'
+prePushFile='.git/hooks/pre-push'
+gitFolder='.git'
+
+if [ ! -d $gitFolder ]
+then
+    echo 'git not initialized'
+else
+    if [ -f $preCommitFile ] && [ -z $GITHUB_ACTIONS ]
+    then
+        rm $preCommitFile
+    fi
+
+    if [ -f $prePushFile ] && [ -z $GITHUB_ACTIONS ]
+    then
+        rm $prePushFile
+    fi
+
+    if [ ! -f $preCommitFile ] && [ -z $GITHUB_ACTIONS ]
+    then
+        curl https://thoughtworks.github.io/talisman/install.sh > ~/install-talisman.sh
```

Comment:
@swatigoel we are downloading talisman install file from thoughtworks github.

this should fine right? if we don't want to do this then we have store shell script in our repo

Reply:
Let it be downloading from thoughtworks github. It's open source, so should not be worry